src/ssl.c

```diff
@@ -3448,6 +3448,208 @@ void wolfSSL_CertManagerFree(WOLFSSL_CERT_MANAGER* cm)
 
 }
 
+#if defined(OPENSSL_EXTRA) && !defined(NO_FILESYSTEM)
+#if defined(WOLFSSL_SIGNER_DER_CERT)
```

Comment:
Can you make sure and add some documentation / code comments for use of `WOLFSSL_SIGNER_DER_CERT` and also submit a PR to testing repo for adding test case on this?

Reply:
Added docs as a function header comments.